### ISSUE TITLE
How to load the model in another file?

### ISSUE THREAD

**Issue by EmreTokyuez** [bentrevett/pytorch-sentiment-analysis#84]
I followed your Upgraded Model Tutorial (and the others of course) but I can't seem to get it working with flask.

What do I need to save (vocab, padding?) and how do I do that? Pickle?

How can I use the model in another py document? Copy the model class? I can't seem to get it working even though I looked at some tutorials online.

**Comment by asvskartheek** [comment]
Assuming that you're trying to make this production-ready, you should've saved:
1. The model to some checkpoint (contains the trained weights, check the PyTorch documentation of saving and loading models)
2. The model hyperparameters (usually saved to YAML files)
3. The vocab of TEXT field (Refer to this [previous issue](https://github.com/bentrevett/pytorch-sentiment-analysis/issues/40)

Coming to the second part, why do you need to copy the class? You can just import it, from that particular file.
